conjur.gemspec

```diff
@@ -17,7 +17,7 @@ Gem::Specification.new do |gem|
 
 
   gem.add_dependency 'activesupport'
-  gem.add_dependency 'conjur-api', '~> 4.20', '>= 4.20.1'
+  gem.add_dependency 'conjur-api', '>= 4.20.3'
```

Comment:
The third version is the "patch" version, which only indicates bug fixes. What we're trying to do here is make sure that the API has the features that the CLI needs, so only the first two numbers are important. Also, the API version can simply be 4.20.0, because we haven't released it to Rubygems yet (so we can keep adding stuff).